src/main/kotlin/org/team5419/fault/subsystems/Subsystem.kt

```diff
@@ -0,0 +1,45 @@
+package org.team5419.fault.subsystems
+
+import org.team5419.fault.util.loops.ILooper
+import java.util.concurrent.atomic.AtomicLong
+
+object SubsystemManager {
```

Comment:
Do we need a way to get subsystems? Are we just going to pass all of them around manually? How is this going to work?